bin/ssm

```diff
@@ -118,6 +112,11 @@ addNonTestJarInDir "${SMART_HOME}/smart-server/target"
 addNonTestJarInDir "${SMART_HOME}/smart-agent/target"
 addJarInDir "${SMART_HOME}/lib"
 
+if [ $SSM_VERSION = "true" ]; then
+  java -cp ${SMART_HOME}/smart-common/target/classes/ org.smartdata.versioninfo.VersionInfoRead
```

Comment:
java -cp "${SMART_CLASSPATH}" org.smartdata.versioninfo.VersionInfoRead. Try to just use this command, because jars in common has been added to classpath. You need to make sure it works in /ssm/smart-dist/target/.